help.txt

```diff
@@ -14,6 +14,8 @@ Available options:
         The amount of requests to make before exiting the benchmark. If set, duration is ignored.
   -S/--socketPath
         A path to a Unix Domain Socket or a Windows Named Pipe. A URL is still required in order to send the correct Host header and path.
+  --on-port PATH
+        Start the command listed after -- on the command line. When it starts listening on a port, start sending requests to that port with the given URL path.
```

Comment:
We might want to move `PATH` out from this option. See #146.

Reply:
yeah when i wrote that line my understanding of how it would work was wrong. it's a boolean flag in the code and PATH is just the URL. will update :+1: